src/TextUI/Command.php

```diff
@@ -1133,6 +1160,7 @@ protected function showHelp(): void
   --include-path <path(s)>    Prepend PHP's include_path with given path(s)
   -d key[=value]              Sets a php.ini value
   --generate-configuration    Generate configuration file with suggested settings
+  --cache-result-file==<FILE> Specify result cache path and filename
```

Comment:
What about "Specify path to result cache file"?

Reply:
It contains a the full `$path . $filename` not just a path. The filename itself doesn't have to be `.phpunit.result.cache`